t/14-isotovideo.t

```diff
@@ -7,8 +7,17 @@ use Test::Warnings ':report_warnings';
 use Test::MockModule;
 use FindBin '$Bin';
 use lib "$Bin/../external/os-autoinst-common/lib";
-use OpenQA::Test::TimeLimit '20';
 use autodie ':all';
+
+# Each full isotovideo invocation (not using _exit_after_schedule=1)
+# is expected to take maximum ~6.67 seconds per run.
+#
+# There are 3 invocations, that approxs to ~20 seconds.
+#
+# Update this number if a new test involving a full isotovideo invocation is
+# added.
```

Comment:
I would say that's also rather obvious and the comment is redundant considering that we use the TimeLimit in most test modules so the same applies to all: "Update the number if tests need to take longer". IMHO no need for this comment

Reply:
Agree to strongly disagree but check the latest commit where this is removed.